native/cocos/engine/Engine.cpp

```diff
@@ -70,6 +71,12 @@
 #endif
 #include "profiler/Profiler.h"
 
+#if !CC_EDITOR && (CC_PLATFORM == CC_PLATFORM_ANDROID || CC_PLATFORM == CC_PLATFORM_WINDOWS || CC_PLATFORM == CC_PLATFORM_MAC_OSX || CC_PLATFORM == CC_PLATFORM_MAC_IOS)
```

Comment:
Using `CC_PLATFORM_MACOS` instead of `CC_PLATFORM_MAC_OSX`
`CC_PLATFORM_IOS` instead of `CC_PLATFORM_MAC_IOS`.

CC_PLATFORM_MAC_OSX and CC_PLATFORM_MAC_IOS were deprecated.

Reply:
Fixed